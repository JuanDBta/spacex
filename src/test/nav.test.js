import { render } from '@testing-library/react';
import {
  MemoryRouter, Routes, Route,
} from 'react-router-dom';
import NAVBAR from '../components/Navbar';

describe('test navabar', () => {
  test('it should match the snapshoot', () => {
    const { container } = render(
      <MemoryRouter initialEntries={['/Navbar']}>
        <Routes>
          <Route path="/Navbar" element={<NAVBAR />} />
        </Routes>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
