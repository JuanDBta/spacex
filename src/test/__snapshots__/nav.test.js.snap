// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test navabar it should match the snapshoot 1`] = `
<div>
  <nav
    class="navbar"
  >
    <li>
      <a
        class=""
        href="/"
      >
        Rockets
      </a>
    </li>
    <li
      class="missions"
    >
      <a
        class=""
        href="/Mission"
      >
        Missions
      </a>
    </li>
    <li>
      <a
        class=""
        href="/Profile"
      >
        My Profile
      </a>
    </li>
  </nav>
</div>
`;
